packages/compass-serverstats/src/components/index.tsx

```diff
@@ -57,8 +63,10 @@ function PerformanceComponent() {
     };
   }, []);
 
+  const { className: scrollbarsClassName } = useScrollbars();
+
   return (
-    <section className="rt-perf">
+    <section className={`rt-perf ${scrollbarsClassName}`}>
```

Comment:
Why are we explicitly applying these styles here? I thought this component is inside compass-home and shouldn't need these 🤔